substrate/frame/identity/src/types.rs

```diff
@@ -241,17 +246,8 @@ pub trait IdentityInformationProvider:
 	/// Check if an identity registered information for some given `fields`.
 	fn has_identity(&self, fields: u64) -> bool;
```

Comment:
```suggestion
	/// Check if an identity registered information for some given `fields`.
	fn has_identity(&self, fields: Self::IdentityField) -> bool;
```

`IdentityField` should get a better name.

Reply:
@joepetrowski suggested `FieldsBitFlags` and I don't have a better idea :sweat_smile: